packages/redux/src/settings/types/action.types.ts

```diff
@@ -0,0 +1,62 @@
+import type * as actionTypes from '../actionTypes';
+import type { Action } from 'redux';
+import type {
+  BlackoutError,
+  Configuration,
+  Configurations,
+} from '@farfetch/blackout-client';
+import type { NormalizedSchema } from 'normalizr';
+
+type Payload = NormalizedSchema<
+  { configurations: Record<Configuration['code'], Configuration> },
+  Configurations
+>;
+
+export interface FetchConfigurationsRequestAction extends Action {
+  type: typeof actionTypes.FETCH_CONFIGURATIONS_REQUEST;
+}
+export interface FetchConfigurationsSuccessAction extends Action {
+  type: typeof actionTypes.FETCH_CONFIGURATIONS_SUCCESS;
+  payload: Payload;
+}
+export interface FetchConfigurationsFailureAction extends Action {
+  type: typeof actionTypes.FETCH_CONFIGURATIONS_FAILURE;
+  payload: { error: BlackoutError };
+}
+
+/**
+ * Actions dispatched when the fetch configurations request is made.
+ */
+export type FetchConfigurationsAction =
+  | FetchConfigurationsRequestAction
+  | FetchConfigurationsSuccessAction
+  | FetchConfigurationsFailureAction;
+
+export interface FetchConfigurationRequestAction extends Action {
+  meta: { code: Configuration['code'] };
+  type: typeof actionTypes.FETCH_CONFIGURATION_REQUEST;
+}
+export interface FetchConfigurationSuccessAction extends Action {
+  meta: { code: Configuration['code'] };
+  type: typeof actionTypes.FETCH_CONFIGURATION_SUCCESS;
+  payload: Payload;
```

Comment:
I believe the Payload type is not correct here as the result property is different. Here it will receive a single Configuration object instead of an array. I think it is better if you create another type just for the FetchConfigurationSuccessAction instead of reusing the same type used by the FetchConfigurationsSuccessAction.